rotkehlchen/tests/unit/test_globaldb_upgrades.py

```diff
@@ -1,64 +1,136 @@
+            if name == 'Aurora DAO':
+                assert swapped_for == _old_strethaddress_to_identifier('0xB705268213D593B8FD88d3FDEFF93AFF5CbDcfAE')  # noqa: E501
+
+            if swapped_for and swapped_for not in ('AM', 'PHB', 'FIRO', 'DIVI', 'SCRT', 'HAI', 'MED', 'NOAHP', 'VET', 'XDC'):  # noqa: E501
+                assert swapped_for.startswith(ETHEREUM_DIRECTIVE)
+
+        # Check some swapped for that we know should have changed. DIVX -> DIVI
+        cursor.execute('SELECT swapped_for from ASSETS where identifier=?', (_old_strethaddress_to_identifier('0x13f11C9905A08ca76e3e853bE63D4f0944326C72'),))  # noqa: E501
+        assert cursor.fetchone()[0] == 'DIVI'
+        # GNT -> GLM
+        cursor.execute('SELECT swapped_for from ASSETS where identifier=?', (_old_strethaddress_to_identifier('0xa74476443119A942dE498590Fe1f2454d7D4aC0d'),))  # noqa: E501
+        assert cursor.fetchone()[0] == _old_strethaddress_to_identifier('0x7DD9c5Cba05E151C895FDe1CF355C9A1D5DA6429')  # noqa: E501
+
+        # Make sure the number of assets remained the same
+        cursor = globaldb.conn.cursor()
+        assert cursor.execute('SELECT COUNT(*) from assets').fetchone()[0] == 1886
+        assert cursor.execute('SELECT COUNT(*) from user_owned_assets').fetchone()[0] == 105
+        # Make sure that populated underlying assets are still there
+        query = cursor.execute('SELECT * from underlying_tokens_list;')
+        assert query.fetchall() == [
+            ('0x42Fa37aC7c115bf17ca5DDfcb94b73b91B10B61B', '0.5', '0xBBc2AE13b23d715c30720F079fcd9B4a74093505'),  # noqa: E501
+            ('0x647C4CD779043b3f00a4ccdec550F35Dd18792b3', '0.5', '0xBBc2AE13b23d715c30720F079fcd9B4a74093505'),  # noqa: E501
+        ]
+        # Make sure that the previous custom assets are still in the DB
+        query = cursor.execute(
+            'SELECT COUNT(*) from assets where identifier IN (?, ?, ?, ?);',
+            (
+                '_ceth_0x35bD01FC9d6D5D81CA9E055Db88Dc49aa2c699A8',
+                '_ceth_0xBBc2AE13b23d715c30720F079fcd9B4a74093505',
+                '_ceth_0x42Fa37aC7c115bf17ca5DDfcb94b73b91B10B61B',
+                '_ceth_0x647C4CD779043b3f00a4ccdec550F35Dd18792b3',
+            ),
+        )
+        assert query.fetchone()[0] == 4
+        query = cursor.execute(
+            'SELECT COUNT(*) from ethereum_tokens where address IN (?, ?, ?, ?);',
+            (
+                '0x35bD01FC9d6D5D81CA9E055Db88Dc49aa2c699A8',
+                '0xBBc2AE13b23d715c30720F079fcd9B4a74093505',
+                '0x42Fa37aC7c115bf17ca5DDfcb94b73b91B10B61B',
+                '0x647C4CD779043b3f00a4ccdec550F35Dd18792b3',
+            ),
+        )
+        assert query.fetchone()[0] == 4
+
+
+@pytest.mark.parametrize('globaldb_version', [2])
+@pytest.mark.parametrize('target_globaldb_version', [3])
+def test_upgrade_v2_v3(globaldb):
+    # at this point upgrade should have happened
```

Comment:
What does this comment even mean? You seem to have copied it without thinking from the previous test. Probably a docstring is better, which should say that thanks to the fixture, at the start of the test we are at the already upgraded DB

But this is not a nice way of testing actually.

As we do in the user DB we should load the old DB, assert it uses the old format and then perform the upgrade and then assert things changed.

Reply:
It was there from the copy but just to acknowledge the same information. The upgrade of the database since the globaldb object is created was made.